hoomd_validation/simple_polygon.py

```diff
@@ -0,0 +1,652 @@
+# Copyright (c) 2022-2023 The Regents of the University of Michigan.
+# Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+"""Hard disk equation of state validation test."""
+
+from config import CONFIG
+from project_class import Project
+from flow import aggregator
+import util
+import os
+import json
+import pathlib
+
+# Run parameters shared between simulations.
+# Step counts must be even and a multiple of the log quantity period.
+RANDOMIZE_STEPS = 20_000
+EQUILIBRATE_STEPS = 100_000
+RUN_STEPS = 2_500_000
+RESTART_STEPS = RUN_STEPS // 10
```

Comment:
```suggestion
RESTART_STEPS = RUN_STEPS // 50
```
Shorten the restart period from ~30 minutes down to ~6 (tested on cheme-hodges).